lxd/db/cluster/permissions.go

```diff
@@ -92,3 +94,32 @@ func GetPermissionEntityURLs(ctx context.Context, tx *sql.Tx, permissions []Perm
 
 	return validPermissions, danglingPermissions, entityURLs, nil
 }
+
+// DeletePermissions deletes the given permissions.
+func DeletePermissions(ctx context.Context, tx *sql.Tx, danglingPermissions []Permission) error {
```

Comment:
Do we need this function anymore?

Reply:
No we don't. The linter should have complained about this as it's unused :thinking: